microshift_storage/microshift-storage-overview.adoc

```diff
@@ -0,0 +1,35 @@
+:_content-type: ASSEMBLY
```

Comment:
You might want to name this file index.adoc, so that it is the landing page for this whole section.